doc/INSTALL-WINDOWS

```diff
@@ -16,11 +16,12 @@ system.
 
     - Open the Command Prompt in Windows (see [1])
 
-    - Install required CygWin packages.
-      - C:\cygwin64\setup-x86_64.exe -q -P gcc-core -P libgcc1 -P make -P perl
-      - C:\cygwin64\setup-x86_64.exe -q -P libssl-devel -P libbz2-devel
-      - C:\cygwin64\setup-x86_64.exe -q -P libgmp-devel -P zlib-devel
-      - C:\cygwin64\setup-x86_64.exe -q -P libOpenCL-devel -P libcrypt-devel
+    - Install required CygWin packages from where you have the CygWin setup:
+      - cd C:\Users\Name\Downloads...
```

Comment:
I'm unhappy about suggesting that people run the setup program from the downloads directory, because this is vulnerable to past drive-by DLL download attacks.